rebar.config

```diff
@@ -5,3 +5,19 @@
 {profiles, [
     {test, [{erl_opts, [nowarn_export_all]}]}
 ]}.
+
+%% the plugin itself
+{plugins, [rebar3_proper]}.
+%% The PropEr dependency is required to compile the test cases
+%% and will be used to run the tests as well.
+{profiles,
+    [{test, [
+        {deps, [
+            %% hex
+            {proper, "1.3.0"},
+            %% newest from master
+            {proper, {git, "https://github.com/proper-testing/proper.git",
```

Comment:
Drop this one, we'll only use the one from hex

Reply:
Done.